packages/ffe-searchable-dropdown-react/src/SearchableDropdown.js

```diff
@@ -264,8 +274,15 @@ SearchableDropdown.propTypes = {
     inputId: isRequiredIf(string, props => props.label),
     /** Value of input field */
     inputValue: string,
-    /** value to be displayed in dropdown in case of search with no match */
-    noMatch: string.isRequired,
+    noMatch: oneOfType([
+        /** value to be displayed in dropdown in case of search with no match */
+        string.isRequired,
```

Comment:
I think you can remove `isRequired` here

Reply:
Then I'll clean up the proptypes and it should be good to go.